### DISCUSSION TITLE
Release Safari extension outside of the App Store

### DISCUSSION THREAD

**Discussion by BayLee4** [simple-login/app#2085] [Feature Requests]
Users without an AppleID cannot use the extension at the moment. AppleID which apparently cannot be created using a SL alias (the account is immediately flagged).

**Comment by ntnhon** [comment]
Hi, we will support dmg files as an alternative to App Store. Will keep you posted.

**Reply by BayLee4** [reply]
That is a very welcoming news @ntnhon, thank you!

**Reply by BayLee4** [reply]
Any update?
